bazel/external/requirements.in

```diff
@@ -47,6 +47,7 @@ routes==2.5.1
 tinyrpc==1.1.4
 webob==1.8.7
 ovs==2.16.0
+sdnotify>=0.3.2
```

Comment:
Would it make sense to add a `,<1` to avoid an unintentional major version upgrade?